.devcontainer/localstack/init.sh

```diff
@@ -17,6 +17,19 @@ corsConfig='{
                 "null"
             ],
             "ExposeHeaders": []
+        },
+        {
+            "AllowedHeaders": [
+                "*"
+            ],
+            "AllowedMethods": [
+                "HEAD",
+                "GET",
+            ],
+            "AllowedOrigins": [
+                "*"
+            ],
+            "ExposeHeaders": []
         }
```

Comment:
What is the difference between this and the above ? Out of curiosity.

Reply:
The rule above allow only origins `null` to manipulate the files (in our case the backend). 
The second rule allow any origin to get the files, important for apps to fetch files.
Note: you can only access files given a signed url, given by the backend.

Reply:
OK ! 🔥